dev_tools/docs/Dockerfile

```diff
@@ -26,8 +26,9 @@
 # PR.
 ########################################################################################
 
-FROM nikolaik/python-nodejs:python3.7-nodejs14
+FROM nikolaik/python-nodejs:python3.9-nodejs16
```

Comment:
Please revert all Dockerfile-s and submit their updates as a new PR.
The same for cirq-all-no-contrib.txt unless it is needed for ci-daily.

It is better if we have PRs focused to a single topic only, it makes them easier to review and/or to revert should they introduce any problem.